### DISCUSSION TITLE
Removing a value from a tag without replacing the entire tag

### DISCUSSION THREAD

**Discussion by Desynq** [FabricMC/fabric-api#2329] [Mod Dev Support] [answered]
I'm trying to make lapis lazuli ore mineable using only a iron tools or better instead of stone tools or better.

I understand that you can append to tags by using the `"replace": false` key, value pair.
I am wondering how you can remove a value from a tag without having to use `"replace": true` and therefore having to manually add every other value previously present in the tag.

**Comment by warjort** [comment, marked as answer]
The vanilla minecraft resource loader is not finegrained enough to do this.

There is kubejs that lets you play around with tags after they are loaded using javascript (not tried this method myself)
https://mods.latvian.dev/books/kubejs/page/tageventjs

Or you can always do something similar to its mixin
https://github.com/KubeJS-Mods/KubeJS/blob/1.18/main/common/src/main/java/dev/latvian/mods/kubejs/mixin/common/TagLoaderMixin.java
the modification of map after the "load" is complete.

**Comment by partisani** [comment]
Hey, i think i found a solution.

If you get the tag's entries, you can save them, remove a value, and place it back again
